### DISCUSSION TITLE
Extend Geo Vocabulary

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1312] [Dev] [answered]
I want to extend the Geo vocabulary

https://github.com/apache/streampipes/blob/dev/streampipes-vocabulary/src/main/java/org/apache/streampipes/vocabulary/Geo.java

with 
1) Geometry Types
https://opengeospatial.github.io/ogc-geosparql/geosparql11/sf_geometries.html and 
Point, Line, Polygon, MultiPoint, MultiLine MultiPolygon and GeometryCollection

There is a hierarchical order, but this could be handled if semantic Type List is harmonized https://github.com/apache/streampipes/issues/1243#issue-1576805363 

2) CRS and/or EPSG Number and additional information
e.g.
http://www.opengis.net/def/crs/EPSG/9.9.1/4326
This part could change during runtime due to Reprojection 

or CRS in general:
http://data.ign.fr/def/ignf#CartesianCS

What do you think?

**Comment by bossenti** [comment, marked as answer]
Sounds plausible for me 👍🏼
